README.md

```diff
@@ -1,3 +1,18 @@
+This package is abandoned
+=========================
+
+This package is now abandoned and will not receive further updates. The underlying transliteration
+dataset it is based on has not been updated since 2016, and there are now better options available.
+Additionally, there are some complications around licensing (see #3) due to the mixed licenses of
```

Comment:
this should use a proper link as `#3` in a readme does not create a reference to the issue.

Reply:
Thanks, good spot.